source/jormungandr/jormungandr/customrediscache.py

```diff
@@ -0,0 +1,170 @@
+from flask_caching._compat import integer_types, string_types
+from flask_caching.backends.base import BaseCache, iteritems_wrapper
+
+try:
+    import cPickle as pickle
+except ImportError:  # pragma: no cover
+    import pickle  # type: ignore
+
+
+class CustomRedisCache(BaseCache):
+    """Uses the Redis key-value store as a cache backend.
+
+    The first argument can be either a string denoting address of the Redis
+    server or an object resembling an instance of a redis.Redis class.
+
+    Note: Python Redis API already takes care of encoding unicode strings on
+    the fly.
+
+    :param write_client, read_client : address of the Redis server or an object which API is
+                 compatible with the official Python Redis client (redis-py).
+    :param port: port number on which Redis server listens for connections.
+    :param password: password authentication for the Redis server.
+    :param db: db (zero-based numeric index) on Redis Server to connect.
+    :param default_timeout: the default timeout that is used if no timeout is
+                            specified on :meth:`~BaseCache.set`. A timeout of
+                            0 indicates that the cache never expires.
+    :param key_prefix: A prefix that should be added to all keys.
+
+    Any additional keyword arguments will be passed to ``redis.Redis``.
+    """
+
+    def __init__(
+        self,
+        write_client="localhost",
+        read_client="localhost",
+        port=6379,
+        password=None,
+        db=0,
+        default_timeout=300,
+        key_prefix=None,
+        **kwargs
+    ):
+        super(CustomRedisCache, self).__init__(default_timeout)
+        if write_client is None and read_client is None:
```

Comment:
Should this be an `OR`? Do you want to continue is either of them is `None`.
```suggestion
        if write_client is None or read_client is None:
```

Or maybe, this can just be only if `write_client` is `None` since `read_client` is replaced by `write_client` below...
```suggestion
        if write_client is None:
```

Reply:
:+1: